libMBIN/Source/Template/NMSTemplate.cs

```diff
@@ -199,7 +199,17 @@ public class NMSTemplate
                         return fieldType == "Int32" ? (object)reader.ReadInt32() : (object)reader.ReadUInt32();
                     }
 
-                    if (field.IsArray) {
+					if( typeof(IEnumArray).IsAssignableFrom(field) ) {
```

Comment:
your indentation is a bit... crazy? haha

Reply:
Doesn't look like that in my VS.  Sure it's not just line mismatch in diff viewer ?  I use tabs at the start of lines, is that it ?

Reply:
Ah, I remember, GitHub defaults to 8 char for each tab.  I use a .editorconfig file w/ the following:
[*]
indent_style = tab
indent_size = 4
trim_trailing_whitespace = true

add that to the project and indentation will look normal in GitHub